resources/assets/css/_general.css

```diff
@@ -229,3 +229,10 @@ body {
     top: 0;
     width: 100%;
 }
+
+/* Child text wrapping */
+@variants responsive {
+    .no-wrap-children > span {
```

Comment:
should we make it clear it's only `span` children? 🤔

Reply:
renamed to `no-wrap-span-children`

Reply:
@ItsANameToo